server/routes/trace_analytics_dsl_router.ts

```diff
@@ -16,21 +16,28 @@ import {
   TRACE_ANALYTICS_JAEGER_INDICES_ROUTE,
 } from '../../common/constants/trace_analytics';
 import { addRequestToMetric } from '../common/metrics/metrics_helper';
+import { getTenantIndexName } from '../../common/utils/tenant_index_name';
 
 export function registerTraceAnalyticsDslRouter(router: IRouter, dataSourceEnabled: boolean) {
   router.post(
     {
       path: TRACE_ANALYTICS_DATA_PREPPER_INDICES_ROUTE,
       validate: {
-        body: schema.any(),
+        body: schema.object({
+          tenant: schema.maybe(schema.string()),
```

Comment:
Will this be a problem to add a hard enforced schema in as compared to `any` before. This is because with this change we aren't other fields passed in body apart from tenant today.

Reply:
i didn't understand the question?

Reply:
Does the body contain other fields and should they be added in the schema?